### ISSUE TITLE
Relaxed versions?

### ISSUE THREAD

**Issue by daveisfera** [schrodinger/fixed-data-table-2#500]
## Expected Behavior
`^` is used to specify versions of dependencies

## Current Behavior
Specific versions are used:
```
prop-types "15.7.2"
redux "4.0.1"                                                                                      
reselect "4.0.0"
```

## Possible Solution
Start using `^` in the `package.json` file

**Comment by wcjordan** [comment]
Thanks for reaching out, and sorry this slipped through the cracks.  We'd be happy to accept a PR for this.

**Comment by wcjordan** [comment]
This is now released w/ v1.1.
